src/VisualStudio/Core/Def/CodeLens/CodeLensCallbackListener.cs

```diff
@@ -125,10 +160,24 @@ public async Task<ImmutableDictionary<Guid, string>> GetProjectVersionsAsync(Imm
 
     public async Task<ImmutableArray<ReferenceMethodDescriptor>?> FindReferenceMethodsAsync(
         CodeLensDescriptor descriptor, CodeLensDescriptorContext descriptorContext, CancellationToken cancellationToken)
```

Comment:
Why not just keep this as the contract for UT as well? CodeLens is the only use case - so a more general API for UT does not seem like it would have significant other benefits.

Reply:
discussed above.